src/app/pages/things/channels/channels.component.ts

```diff
@@ -162,4 +133,43 @@ export class ChannelsComponent implements OnInit {
       },
     );
   }
+
+  onFileSelected(files: FileList) {
```

Comment:
Is this the function for uploading data for creation OPC-UA nodes?
If so, I think it would be much better to give the possibilty to add CSV data instead of metadata.
Look at the files we got from the machine producer.

Reply:
No, this is for creation of Things and Channels. You can specify the name and the metadata in JSON. For the OPC-UA nodes we need also the implementation of connections bulk endpoint